### PR TITLE
Update the Schema Status to lowercase to be consistent with the statu…

### PR DESCRIPTION
**Pull request by apurvabhaleMS**
## Description
Update the Schema Status to lowercase to be consistent with the statuses applied when AutomaticUpdates is turned on

## Related issues
Addresses [issue [81910](https://microsofthealth.visualstudio.com/Health/_workitems/edit/81910)].

## Testing
Manual Testing

## Semver Change ([docs](https://github.com/microsoft/healthcare-shared-components/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking

### CODE REVIEW

File: src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs [hunk 1]
```diff
@@ -11,8 +11,16 @@ namespace Microsoft.Health.SqlServer.Features.Schema
     [JsonConverter(typeof(StringEnumConverter))]
     public enum SchemaVersionStatus
     {
-        Started,
-        Completed,
-        Failed,
+        // Note - Updating SchemaVersionStatus to lower case to be consistent with the statuses applied when schema AutomaticUpdates is set to true
```
Comment (rbans96) [inline src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs:14]: nit: schema AutomaticUpdates -> SchemaOptions:AutomaticUpdatesEnabled?

File: src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs [hunk 2]
```diff
@@ -11,8 +11,16 @@ namespace Microsoft.Health.SqlServer.Features.Schema
...
+
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+        started,
+#pragma warning restore SA1300 // Element should begin with upper-case letter
```
Comment (rbans96) [inline src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs:18]: I am wondering, if we could have warning disable at the top and warning restore at the bottom and avoid the intermediate ones?

Comment (apurvabhaleMS) [reply src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs]: just updated

File: src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs (hunk 1 above)
Comment (apurvabhaleMS) [reply src/Microsoft.Health.SqlServer/Features/Schema/SchemaVersionStatus.cs]: Yep! Added the exact name

**Review by rbans96 — APPROVED** [review]

**Pull request merged**